kube-client/src/config/file_config.rs

```diff
@@ -262,6 +256,13 @@ impl Kubeconfig {
         Ok(config)
     }
 
+    /// Read a Config from arbitrary text
+    pub fn from_text(text: &str) -> Result<Kubeconfig, KubeconfigError> {
```

Comment:
```suggestion
    /// Read a Config from arbitrary YAML document
    pub fn from_yaml_str(text: &str) -> Result<Kubeconfig, KubeconfigError> {
```

Reply:
Probably also want to add a doccomment about why this is preferred over `serde_yaml::from_str`

Reply:
Good point, will do

Reply:
Done